scipy/signal/wavelets.py

```diff
@@ -330,6 +411,14 @@ def cwt(data, wavelet, widths):
         satisfies these requirements.
     widths : (M,) sequence
         Widths to use for transform.
+    dtype : data-type, optional
+        The desired data type of output. Defaults to ``float64`` if the
+        output of `wavelet` is real and `data` is real. If either one of
+        them is complex, `dtype` is set to ``complex128``.
+        .. versionadded:: 1.1.0
```

Comment:
You need a newline before this `..` directive, see the failed render:

https://4416-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.signal.cwt.html#scipy.signal.cwt